README.md

```diff
@@ -159,3 +159,107 @@ spec:
         - "--dns01-recursive-nameservers=1.1.1.1:53"
         - "--dns01-recursive-nameservers-only"
 ```
+## Enabling metrics and monitoring for `cert-manager`
+
+Cert-Manager exposes controller metrics in the format expected by [Prometheus Operator](https://github.com/prometheus-operator/prometheus-operator). 
+
+We will be using the `ServiceMonitor` object in the operand's namespace (by default `cert-manager`), make sure that your instance of the Prometheus Operator is properly configured to find it.
+
+You can check `.spec.serviceMonitorNamespaceSelector` and `.spec.serviceMonitorSelector` fields of `prometheus` resource.
+
+```sh
+kubectl -n monitoring get prometheus k8s --template='{{.spec.serviceMonitorNamespaceSelector}}{{"\n"}}{{.spec.serviceMonitorSelector}}{{"\n"}}'
+map[matchLabels:map[openshift.io/cluster-monitoring:true]]
+map[]
+```
+For OpenShift:
+```sh
+oc -n openshift-monitoring get prometheus k8s --template='{{.spec.serviceMonitorNamespaceSelector}}{{"\n"}}{{.spec.serviceMonitorSelector}}{{"\n"}}'
+map[matchLabels:map[openshift.io/cluster-monitoring:true]]
+map[]
+```
+Lable the Operand's namespace to enable the cluster monitoring.
+
+`
+$ oc label namespace cert-manager openshift.io/cluster-monitoring=true
+`
+
+Please follow the steps below to `enable the monitoring for user-defined projects` in Openshift:
+
+Cluster administrators can enable monitoring for user-defined projects by setting the `enableUserWorkload: true` field in the cluster monitoring ConfigMap object.
+
+1. Edit the cluster-monitoring-config ConfigMap object:
```

Comment:
nit: Is it ok to skip the commands to oc edit and prescribe only the config data?

Reply:
that's a really important step, so I let it in a detailed way to make it easier for the end-user of this documentation.